<!-- -->
<launch>
    <arg name='host' default='localhost'/>
    <arg name='port' default='2000'/>
    <arg name='number_of_vehicles' default='10'/>
    <arg name="role_name" default="ego_vehicle"/>
    <param name="carla/host" value="$(arg host)" unless="$(eval host == '')"/>
    <param name="carla/port" value="$(arg port)" unless="$(eval port == '')"/>
    <param name="carla/npcs" value="$(arg number_of_vehicles)" unless="$(eval port == '')"/>
    <node pkg="carla_ros_spawn_npc" type="carla_ros_spawn_npc.py" name="carla_ros_spawn_npc" output="screen">
    </node>
</launch>